libs/ts/contracts/tsconfig.json

```
@@ -10,6 +10,6 @@
     "outDir": "dist",
     "rootDir": "."
   },
-  "include": ["**/*.ts"],
+  "include": ["**/*.ts", "abis/**/*.json"],
```

This change doesn't seem to be necessary:

```suggestion
  "include": ["**/*.ts"],
```